src/core/timer.c

```diff
@@ -893,6 +906,64 @@ static int timer_can_start(Unit *u) {
         return 1;
 }
 
+static void activation_event_info_timer_serialize(ActivationEventInfo *info, FILE *f) {
+        ActivationEventInfoTimer *t = ACTIVATION_EVENT_INFO_TIMER(info);
+
+        assert(info);
+        assert(f);
+        assert(t);
+
+        (void) serialize_dual_timestamp(f, "activation-event-info-timer-last-trigger", &t->last_trigger);
+}
+
+static int activation_event_info_timer_deserialize(const char *key, const char *value, ActivationEventInfo **info) {
+        int r;
+
+        assert(key);
+        assert(value);
+
+        if (streq(key, "activation-event-info-timer-last-trigger")) {
+                if (!info || !*info)
+                        return -EINVAL;
+
+                ActivationEventInfoTimer *t = ACTIVATION_EVENT_INFO_TIMER(*info);
+                if (!t)
+                        return -EINVAL;
+
+                r = deserialize_dual_timestamp(value, &t->last_trigger);
+                if (r < 0)
+                        return r;
+
+                return 0;
+        }
+
+        return -EINVAL;
+}
+
+static int activation_event_info_timer_format(ActivationEventInfo *info, char ***strv) {
+        ActivationEventInfoTimer *t = ACTIVATION_EVENT_INFO_TIMER(info);
+        char *s;
+
+        assert(info);
+        assert(strv);
+        assert(t);
+
+        if (!dual_timestamp_is_set(&t->last_trigger))
+                return 0;
+
+        s = strjoin("TRIGGER_TIMER_REALTIME=", FORMAT_TIMESTAMP(t->last_trigger.realtime));
+        if (!s)
+                return -ENOMEM;
+
+        if (strv_consume(strv, TAKE_PTR(s)) < 0)
+                return -ENOMEM;
+
+        if (strv_extendf(strv, "TRIGGER_TIMER_MONOTONIC=%" PRIu64, t->last_trigger.monotonic) < 0)
```

Comment:
following discussion above: rename TRIGGER_TIMER_MONOTONIC_USEC?

also, output via USEC_FMT, not PRIu64 please